package.json

```diff
@@ -81,6 +84,13 @@
       "post-rewrite": "yarn run ts-node ./scripts/hooks/post-rewrite.ts"
     }
   },
+  "jest": {
+    "coverageReporters": ["json"]
+  },
+  "nyc": {
```

Comment:
Move to its own configuration file (assuming nyc supports configuration files vs package.json configuration)